.github/workflows/ci.yml

```diff
@@ -50,7 +50,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: "PyAnsys documentation style checks"
+<<<<<<< maint/remove-start_mapdl_ubuntush-file
         uses: pyansys/actions/doc-style@v3.0.4
+=======
+        uses: pyansys/actions/doc-style@v3.0.8
+>>>>>>> main
```

Comment:
Wow... watch out. Something weird happened @clatapie

Reply:
That's very weird indeed, thanks for the warning!

Reply:
It is fixed now.